pkg/apply/error/error.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+package error
+
+type UnknwonTypeError struct {
```

Comment:
I think there is a typo here. It should be `UnknownTypeError` right?

Reply:
fixed.